CMakeLists.txt

```diff
@@ -170,8 +166,8 @@ add_subdirectory(inc)
 set(STLINK_HEADERS
         inc/backend.h
         inc/stlink.h
```

Comment:
not all *.h files listed in this list

Reply:
Note, that we have another CMakeLists.txt in /inc with 

```
file(GLOB STLINK_HEADERS
    "${CMAKE_SOURCE_DIR}/inc/*.h"
    "${CMAKE_SOURCE_DIR}/src/stlink-lib/*.h"
    "${CMAKE_BINARY_DIR}/inc/version.h")
install(FILES ${STLINK_HEADERS} DESTINATION ${STLINK_INCLUDE_PATH})
```

Reply:
@slyshykO you can comment out the whole `set(STLINK_HEADERS ... )` section to verify the correct compilation on your machine. This works for me here. One should update the description/comments in the main CMakeLists.txt however to point out where exactly the header files are included. This is in line 163:
`add_subdirectory(inc)` where the secondary `CMakeLists.txt` is referenced.

Reply:
set(STLINK_HEADERS ... ), it's not about compilation. It's about the consistent representation of the project in various IDE.
Adding header files to target dependency only affects the representation of the project in IDE.
And I suggest adding all headers to this list.

Reply:
All header files for `stlink-lib` have been added.